UI/window-basic-filters.cpp

```diff
@@ -173,6 +174,11 @@ void OBSBasicFilters::UpdatePropertiesView(int row, bool async)
 			(PropertiesReloadCallback)obs_source_properties,
 			(PropertiesUpdateCallback)obs_source_update);
 
+	updatePropertiesSignal.Connect(obs_source_get_signal_handler(filter),
+		"update_properties",
```

Comment:
Style:  Use 2 tabs for indenting line continuations.  See also the next two lines.